sn_networking/src/transfers.rs

```diff
@@ -51,38 +49,14 @@ impl Network {
     /// If we get a quorum error, we increase the RetryStrategy
     pub async fn get_spend(&self, address: SpendAddress) -> Result<SignedSpend> {
         let key = NetworkAddress::from_spend_address(address).to_record_key();
-        let mut get_cfg = GetRecordCfg {
+        let get_cfg = GetRecordCfg {
             get_quorum: Quorum::All,
```

Comment:
Just wondering, what does Quorum::All mean with get range? All the peers that hold the data in the address range? Or all the ones in the close group?

Reply:
You can see [in kad.rs](https://github.com/maidsafe/safe_network/pull/2213/files#diff-4c7753b4198da926db9ea80d1e10daac655d3472dbc84edc034f41a54ffd0022R434-R462) we have `have_we_have_searched_thoroughly_for_quorum` which basically relaxes checks when not `Quorum::All`.

`Spends` were quorum all (as that's where the added security really benefits)